drivers/starburst/resources/metabase-plugin.yaml

```diff
@@ -21,9 +21,11 @@ driver:
           placeholder: tpch
           display-name: Catalog
           required: true
+          helper-text: Starburst Catalogs contain schemas and reference data sources via a connector.
     - name: schema
       display-name: Schema (optional)
       required: false
+      helper-text: Starburst Schemas contain tables
```

Comment:
Can we use the `Only add tables to Metabase that come from a specific schema.`

Reply:
Done